src/transformers/models/bridgetower/configuration_bridgetower.py

```diff
@@ -0,0 +1,403 @@
+    configuration with the defaults will yield a similar configuration to that of the bridgetower-base
+    [BridegTower/bridgetower-base](https://huggingface.co/BridgeTower/bridgetower-base/) architecture.
+
+    Configuration objects inherit from [`PretrainedConfig`] and can be used to control the model outputs. Read the
+    documentation from [`PretrainedConfig`] for more information.
+
+    Args:
+        cross_modal_transform_shared (`bool`, *optional*, defaults to `True`):
+            Whether cross modal transformer layers are shared.
+        drop_rate (`float`, *optional*, defaults to 0.1):
+            Drop out probability.
+        head_hidden_scale (`int`, *optional*, defaults to 2):
+            Scale of hidden layers head.
+        hidden_act (`str` or `function`, *optional*, defaults to `"gelu"`):
+            The non-linear activation function (function or string) in the encoder and pooler.
+        hidden_size (`int`, *optional*, defaults to 768):
+            Dimensionality of the encoder layers and the pooler layer.
+        image_size (`int`, *optional*, defaults to 288):
+            The size (resolution) of each image.
+        input_image_embed_size (`int`, *optional*, defaults to 768):
+            Embedding size of the input image.
+        input_text_embed_size (`int`, *optional*, defaults to 768):
+            Embedding size of the input text.
+        is_encoder_decoder (`bool`, *optional*, defaults to `False`):
+            Whether this is an encoder/decoder model
+        layer_norm_eps (`float`, *optional*, defaults to 1e-05):
+            The epsilon used by the layer normalization layers.
+        link_tower_shared (`bool`, *optional*, defaults to `False`):
+            Whether the bride/link tower layers are shared.
+        link_tower_type (`str`, *optional*, defaults to `"add"`):
+            Type of the bridge/link layer.
+        max_text_len (`int`, *optional*, defaults to 50):
+            Maximum text length.
+        mlp_ratio (`int`, *optional*, defaults to 4):
+            Ratio of MLP hidden dim to embedding dim.
+        num_attention_heads (`int`, *optional*, defaults to 12):
+            Number of attention heads for each attention layer in the Transformer encoder.
+        num_hidden_layers (`int`, *optional*, defaults to 6):
+            Number of hidden layers in the Transformer encoder.
+        text_config (`dict`, *optional*):
+            Dictionary of configuration options used to initialize [`BridgeTowerTextConfig`].
+        tie_word_embeddings (`bool`, *optional*, defaults to `False`):
+            Whether embedding weights are tied with the decoder
+        vision_config (`dict`, *optional*):
+            Dictionary of configuration options used to initialize [`BridgeTowerVisionConfig`].
+        vocab_size (`int`, *optional*, defaults to 50265):
+            Vocabulary size of the text part of the model. Defines the number of different tokens that can be
```

Comment:
Isn't this already part of the text config?

Reply:
Cleaned up the configs. Thanks for the suggestion!